__tests__/app.test.js

```diff
@@ -258,4 +258,52 @@ describe('/api/articles', () => {
 
     })
 
+    describe('POST /api/articles/:article_id/comments', () => {
+
+        test('A comment is added to the appropriate article', () => {
+
+            return request(app)
+                .post('/api/articles/2/comments')
+                .send({ username: 'lurker' , body: 'Comment'})
+                .expect(201)
+                .then((res) => {
+
+                    expect(res.body).toEqual({ commentAdded : 'Comment' })
+
+                })
+
+
+        })
+
+        test('Respond with status of 400 if an invalid data type is used for the id', () => {
+
+            return request(app)
+                .post('/api/articles/wrongdatatype/comments')
+                .expect(400)
+                .then((res) => {
+
+                    expect(res.body.message).toBe('Invalid input.')
+
+                })
```

Comment:
Be careful to include the `.send` with a request body that _is valid_ for this test, otherwise you could be causing a different error to the one you are aiming to test for